### ISSUE TITLE
Cannot install v3.11.0

### ISSUE THREAD

**Issue by adippel** [hairyhenderson/gomplate#1428]
When a try to install latest. which now points to v3.11.0, I get the following error:

```bash
$ go install github.com/hairyhenderson/gomplate/v3/cmd/gomplate@latest
go: github.com/hairyhenderson/gomplate/v3/cmd/gomplate@latest (in github.com/hairyhenderson/gomplate/v3@v3.11.0):
        The go.mod file for the module providing named packages contains one or                                  
        more replace directives. It must not contain directives that would cause                                 
        it to be interpreted differently than if it were the main module.  
```

```
$ go version
go version go1.18.1 windows/amd64
```

Installing v3.10.0 still works.

Is this an issue on your site?

**Comment by hairyhenderson** [comment]
Hi @adippel, sorry to see you ran into this issue... Unfortunately that replace directive is required for now to work around [a bug](https://github.com/go-yaml/yaml/pull/862).

This is a restriction that's been discussed at length [in this Go issue](https://github.com/golang/go/issues/44840), but it seems like there isn't much chance of it being fixed any time soon.

I _could_ avoid it by forking the upstream package and switching to that fork, but IMO the replace directive is a cleaner approach.

Is there a reason you need to install via `go install` rather than downloading a binary or using one of the packages? Obviously `go install` should be a totally fine installation method, but I'm curious 😉

**Comment by mgutz** [comment]
@hairyhenderson `go install` is easier than having to copy download url, wget url, unarchive, move to ~/.local/bin, chmod

Not a big deal.

**Comment by hairyhenderson** [comment]
Thanks for the feedback @mgutz - that's totally fair!

I'll get this solved soon.

**Comment by adippel** [comment]
Totally agree with @mgutz. I use gomplate within a CI Job for building a Go based project. Using go install is just so convenient. 

Thanks for looking into this and for this awesome tool!

**Comment by hairyhenderson** [comment]
FYI this should be resolved by the [`v3.11.1` tag](https://github.com/hairyhenderson/gomplate/releases/tag/v3.11.1). I'm building a release now, but you should already be able to `go install` it.
